resources/config/shapeshifter.php

```diff
@@ -12,6 +12,8 @@
     */
     'customer' => 'Just',
 
+    'breadcrumbs' => true,
```

Comment:
@JustBram Plaese add description above the attribute explaning the configuration (like the rest)